packages/cli/src/constants.js

```diff
@@ -24,9 +23,7 @@ const BLACKLISTED_PATHS = [
   '.environment',
   'build'
 ];
-const nodeVersion = semver.Comparator(
```

Comment:
I have no idea why we needed to do this just to get out the exact same string we were putting in 🤔